DataFormats/simulation/include/SimulationDataFormat/MCTruthContainer.h

```diff
@@ -24,6 +24,7 @@
 #include <cstring> // memmove, memcpy
 #include <memory>
 #include <vector>
+#include <iostream>
```

Comment:
where do we need this header?

Reply:
std::endl

Reply:
Wait. In this case we could replace std::endl with "\n" and not include the header. We should really avoid iostream in header files whenever possible.